### DISCUSSION TITLE
Integrate GraphQL Yoga PubSub bus with TypeGraphQL

### DISCUSSION THREAD

**Discussion by beeirl** [MichalLytek/type-graphql#1451] [Q&A]
I am wondering if there is a way to use the [GraphQL Yoga PubSub bus](https://the-guild.dev/graphql/yoga-server/docs/features/subscriptions#pubsub) as a [custom PubSub system in TypeGraphQL](https://typegraphql.com/docs/subscriptions.html#using-a-custom-pubsub-system). TypeGraphQL requires the custom PubSub integration to implement the `PubSubEngine` interface in order to be used in their `buildSchema` function, which is currently not the case given how Yoga's `createPubSub` function is implemented.

**Comment by MichalLytek** [comment]
What are the differences? Does it work when you put it `as any`?
